docs/resources/csv.md.erb

```diff
@@ -36,12 +36,18 @@ The following examples show how to use this InSpec audit resource.
 
 <br>
 
+## Property Examples
+
+### name
+
+The `name` property tests the value of `name` as read from a CSV file versus the value declared in the test:
```

Comment:
I'm confused by this sentence, specifically "versus the value declared in the test" statement. Can you elaborate?

Reply:
Clarified.